lesson9/main.js

```diff
@@ -0,0 +1,52 @@
+//課題の配列
+const data = [
+   {to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク", src: "img/bookmark.jpeg"},
+   {to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ", src: "img/message.jpeg"}
+];
+
+const body = document.getElementById("body");
+const wrapper = document.createElement("div");
+body.appendChild(wrapper);
+
+//はじめからloading iconを表示するだけなので即時関数にしている
```

Comment:
[IMO]
The role of loading is from the start of data acquisition to the end of acquisition.
So it is better to show and remove loading before and after waitTime().

Reply:
I changed this immediate function to arrow function and moved to before waitTime.
In actual use, it will be executed at the time of data acquisition, so we need to make this kind of specification.
3d108ac